server/db/models/_base.js

```diff
@@ -5,6 +5,14 @@ class BaseModel extends Model {
   static get modelPaths() {
     return [__dirname]
   }
+
+  $beforeInsert() {
```

Comment:
I am not sure if this is necessary; Postgres should automatically add timestamps when you insert a row. Or am I mistaken on what the intent is here?